docs/topics/execute.adoc

```diff
@@ -68,49 +69,49 @@ $ <MTA_HOME>/bin/mta-cli -Dwindup.decompiler=procyon --input <INPUT_ARCHIVE_OR_D
 ----
 
 [discrete]
-[id='cli-bash-completion_{context}']
-== About {ProductShortName} {CLINameTitle} bash completion
+[id="cli-bash-completion_{context}"]
+== About {ProductShortName} {CLINameTitle} Bash completion
 
-The {ProductShortName} {CLIName} provides an option to enable bash completion for Linux systems, allowing the {ProductShortName} command-line arguments to be auto completed by pressing the Tab key when entering the commands. For instance, when bash completion is enabled, entering the following displays a list of available arguments.
+The {ProductShortName} {CLIName} provides an option to enable Bash completion for Linux systems, allowing the {ProductShortName} command-line arguments to be auto completed by pressing the Tab key when entering the commands. For instance, when Bash completion is enabled, entering the following displays a list of available arguments.
 
 [source,options="nowrap",subs="+quotes"]
 ----
 $ <MTA_HOME>/bin/mta-cli [TAB]
 ----
 
 [discrete]
-[id='bash-completion-temporary_{context}']
-=== Enabling bash completion
+[id="bash-completion-temporary_{context}"]
+=== Enabling Bash completion
 
-To enable bash completion for the current shell, execute the following command. After the prompt returns, follow the steps for using the {CLIName}.
+To enable Bash completion for the current shell, execute the following command. After the prompt returns, follow the steps for using the {CLIName}.
```

Comment:
Re: Execute. IBM usage says this for 'execute'
Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.

"After the prompt returns..."??? I think you can remove the second sentence. Fluff. User knows that the prompt will return and can move forward.